spi/src/main/java/org/arquillian/cube/spi/metadata/HasMappedPorts.java

```diff
@@ -0,0 +1,13 @@
+package org.arquillian.cube.spi.metadata;
+
+
+public interface HasMappedPorts extends CubeMetadata {
+    
+    String getIP();
+
+    Integer forTargetPort(int targetPort);
+
+    void createProxies() throws Exception;
```

Comment:
Which means we can kill this class https://github.com/arquillian/arquillian-cube/pull/259/files#diff-2352d8cd76985b787b92036bbf6a7ff1